docs/blueprints/AMD_Secure_Kernel_Loader.md

```diff
@@ -2,97 +2,104 @@
-execution is passed to the user-provided block of code, called Secure Loader in
-AMD documents. To some extent, this code can be treated as the AMD equivalent of
-ACM, except that it doesn't have to be signed by the CPU vendor.
-
-SKINIT extends PCR17 with the hash of SL (Secure Loader), which size is
-specified in the SL header (maximum of 64K - 1), protects SLB (Secure Launch
-Block, always 64K) against DMA access from the devices, puts the CPU in defined
-execution state and jumps to the entry point also specified in the header. At
-the entry point to the SL the CPU works in flat 32-bit protected mode with
-paging disabled. Most general purpose registers are cleared. Interrupts are
-disabled (held pending until re-enabled), including NMI, SMI and INIT. Refer to
+Contrary to the TXT solution, SKINIT is a single CPU instruction after
+which the execution is passed to the user-provided block of code,
+called Secure Loader in AMD documents. To some extent, this code can
+be treated as the AMD equivalent of ACM, except that it doesn't have
+to be signed by the CPU vendor.
+
+SKINIT extends PCR17 with the hash of SL (Secure Loader), which size
+is specified in the SL header (maximum of 64K - 1), protects SLB
+(Secure Launch Block, always 64K) against DMA access from the devices,
+puts the CPU in defined execution state and jumps to the entry point
+also specified in the header. At the entry point to the SL the CPU
+works in flat 32-bit protected mode with paging disabled. Most general
+purpose registers are cleared. Interrupts are disabled (held pending
+until re-enabled), including NMI, SMI and INIT. Refer to
 [AMD APM Vol 2](../references/AMD64-Architecture-Programmers-Manual_Volume-2_Ch15.27.pdf)
 for more details.
 
 Secure Kernel Loader (SKL) is an implementation of Secure Loader.
 
 ## Approach
 
-This is a high-level overview of tasks performed by Secure Kernel Loader:
+This is a high-level overview of tasks performed by Secure Kernel
+Loader:
 
-1. set GDT and segment registers (only CS and SS are valid after SKINIT)
-2. set up pagetables and enable Long Mode (optional, see `BITS` in
+1. set GDT and segment registers (only CS and SS are valid after
+   SKINIT)
+1. set up pagetables and enable Long Mode (optional, see `BITS` in
    [Build options](#build-options))
-3. initialize TPM interface
-4. initialize TPM event log
-    * log what was done by SKINIT
-5. extend PCR18 with the hash of SLRT (Secure Launch Resource Table) passed by
-   the bootloader to the SKL
-    * log that event
-6. obtain information about the kernel from SLRT provided by a bootloader, most
-   notably:
-    * kernel location
-    * kernel size
-    * entry point
-7. extend PCR17 with the hash of the kernel
-    * log that event
-8. set CPU registers according to the kernel boot protocol
-9. jump to the entry point of kernel
+1. initialize TPM interface
+1. initialize TPM event log
+   - log what was done by SKINIT
```

Comment:
Why 3 spaces?

Reply:
I think it's because in both cases it wants the next row to start under the previous one. I think it's fine because it's consistent. But you can try to change that if you know how.

Reply:
Maybe I asked wrong, why 3 spaces instead of 4.

> I think it's fine because it's consistent

It isn't consistent with rest of the documents.

Reply:
Manually fixed in fd82665277fc7d1e920832ee7fb29008484e3470

The pre-commit still passes with the change